docs/api/previous-rancher-api-versions/api-tokens.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Using API Tokens in Past Rancher Versions
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/api/previous-rancher-api-versions/api-tokens"/>
+</head>
+
+By default, some cluster-level API tokens are generated with infinite time-to-live (`ttl=0`). In other words, API tokens with `ttl=0` never expire unless you invalidate them. Tokens are not invalidated by changing a password.
```

Comment:
I think we need to open with a paragraph stating something like, `Rancher v2.8 introduced the [Rancher Kubernetes API](link). With the Rancher Kubernetes API, you can manage Rancher resources through kubectl. In previous versions of Rancher, you could access a Rancher API through your web browser.`

I'm not sure if the web-based API is still accessible through v2.8? If it isn't, we could link to the API intro page in versioned docs for 2.7, aka most recent version where this API was relevant.

Reply:
Added in an intro to this section with appropriate links.